Setup/UpgradeData.php

```diff
@@ -221,5 +229,10 @@ public function upgrade(
                 }
             }
         }
+
+        if (version_compare($context->getVersion(), '1.0.4', '<')) {
```

Comment:
I would add a comment here to explain why this happens for this version.

Reply:
In the UpgradeScheme file, too.

Reply:
@codymullins We don't have comments for any the upgrade steps.  Is there a reason you think this version deserves one?  Or are you suggesting each version should have one?

Reply:
Yeah, it might be good in general - didn't mean "this version especially." For others diagnosing issues and looking through that aren't familiar with the code/history.